### PR TITLE
Adding the ValidateChip pipeline

### PR DESCRIPTION
**Pull request by gbggrant**

### CODE REVIEW

File: pipelines/broad/arrays/validate_chip/ValidateChip.dev.options.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "read_from_cache": true,
```
Comment (jessicaway) [inline pipelines/broad/arrays/validate_chip/ValidateChip.dev.options.json:2]: Do we still need the 3 different options files?

Comment (gbggrant) [reply pipelines/broad/arrays/validate_chip/ValidateChip.dev.options.json:2]: I'm not sure.  Can't remember what google_project is doing there anyways.  I'll see what happens.

Comment (gbggrant) [reply pipelines/broad/arrays/validate_chip/ValidateChip.dev.options.json]: Looks like they're not needed.  Have updated.

Comment (jessicaway) [reply pipelines/broad/arrays/validate_chip/ValidateChip.dev.options.json]: 👍

**Pull request merged**